src/utils/errorSerializer.ts

```diff
@@ -34,61 +38,64 @@ import UserTimeoutError from "../errors/UserTimeoutError";
 import VersionInvalidError from "../errors/VersionInvalidError";
 import VersionMismatchError from "../errors/VersionMismatchError";
 
-const errorCodeToErrorClass: {[key: string]: any} = {
+const errorCodeToErrorClass: Record<keyof typeof ErrorCode, KnownErrorConstructors> = {
   [ErrorCode.ERR_WEBEID_ACTION_PENDING]:          ActionPendingError,
   [ErrorCode.ERR_WEBEID_ACTION_TIMEOUT]:          ActionTimeoutError,
   [ErrorCode.ERR_WEBEID_CONTEXT_INSECURE]:        ContextInsecureError,
   [ErrorCode.ERR_WEBEID_EXTENSION_UNAVAILABLE]:   ExtensionUnavailableError,
+  [ErrorCode.ERR_WEBEID_MISSING_PARAMETER]:       MissingParameterError,
   [ErrorCode.ERR_WEBEID_NATIVE_INVALID_ARGUMENT]: NativeInvalidArgumentError,
   [ErrorCode.ERR_WEBEID_NATIVE_FATAL]:            NativeFatalError,
   [ErrorCode.ERR_WEBEID_NATIVE_UNAVAILABLE]:      NativeUnavailableError,
   [ErrorCode.ERR_WEBEID_USER_CANCELLED]:          UserCancelledError,
   [ErrorCode.ERR_WEBEID_USER_TIMEOUT]:            UserTimeoutError,
   [ErrorCode.ERR_WEBEID_VERSION_INVALID]:         VersionInvalidError,
   [ErrorCode.ERR_WEBEID_VERSION_MISMATCH]:        VersionMismatchError,
+  [ErrorCode.ERR_WEBEID_UNKNOWN_ERROR]:           UnknownError,
 };
 
-export function serializeError(error: any): any {
-  const {
-    message,
-    name,
-    fileName,
-    lineNumber,
-    columnNumber,
-    stack,
-  } = error;
+export function serializeError(error: Error): SerializedError {
+  if (typeof error != "object" || error == null || !(error instanceof Error)) {
+    throw new UnknownError();
```

Comment:
Should you perhaps return an `UnknownError` here instead of throwing?

Reply:
Yes it should.